backend/resources/migrations/080-add-topic-review-table.up.sql

```diff
@@ -0,0 +1,27 @@
+CREATE TYPE topic_name AS ENUM (
+  'stakeholder', 'organisation', 'technology', 'policy', 'resource', 'event', 'initiative'
+);
+
+-- FIXME: Ideally, we would rename the existing review_status type to
+-- approval_status, and rename the field on all the topics/resources,
+-- but this is going to be quite a big rename/refactor to do now.
+
+CREATE TYPE reviewer_review_status AS ENUM (
+  'PENDING', 'ACCEPTED', 'REJECTED'
+);
+
+CREATE TABLE review (
+  id serial NOT NULL PRIMARY KEY,
+  topic_name topic_name NOT NULL,
+  topic_id integer NOT NULL,
+  assigned_by integer NOT NULL REFERENCES stakeholder(id),
+  assigned timestamptz DEFAULT now(),
```

Comment:
it seems strange not use `created` as column name instead of `assigned`... also having all the tables same column name to track db-entity creation-date

Reply:
Fair!